stanford_profile.profile

```diff
@@ -11,6 +11,35 @@ use Drupal\Core\Session\AccountInterface;
 use Drupal\Core\Field\FieldItemListInterface;
 use Drupal\Core\Access\AccessResult;
 
+/**
+ * Implements hook_install_tasks().
+ */
+function stanford_profile_install_tasks(&$install_state) {
+  return ['stanford_profile_final_task' => []];
```

Comment:
These tasks have ids why wouldn't we want them in this weighted array? Perhaps some tasks need to be run before others? How do we control the order of operations?

Reply:
that could be done in the plugin manager

Reply:
Aye, makes sense to put it there.